tests/ui/never_loop.stderr

```diff
@@ -75,6 +75,13 @@ LL | |             _ => return,
 LL | |         }
 LL | |     }
    | |_____^
+   |
+help: if you need the first element of the iterator, try writing
+   |
+LL |     for (0..10).into_iter().next() {
```

Comment:
It should be `if let Some(x) = (0..10).into_iter().next() {`

Reply:
Oh whoops